dashboard/app/controllers/sessions_controller.rb

```diff
@@ -62,6 +64,10 @@ def lockout
     # If the student isn't signed in, go to the login page
     return redirect_to new_user_session_path unless current_user
 
+    # If the user is compliant with the Child Account Policy, redirect them to
+    # /home
+    redirect_to '/home' if Policies::ChildAccount.compliant? current_user
```

Comment:
```suggestion
    return redirect_to home_path if Policies::ChildAccount.compliant? current_user
```

Reply:
Ah thanks, I keep forgetting about the `_path` constants it generates for us.